paasta_tools/setup_kubernetes_job.py

```diff
@@ -140,7 +142,15 @@ def setup_kube_deployments(
             not in existing_apps
         ):
             app.create(kube_client)
-        elif app and app.kube_deployment not in existing_kube_deployments:
+        elif (
+            app
+            and (app.kube_deployment not in existing_kube_deployments)
+            or autoscaling_is_paused()
+        ):
+            app.update(kube_client)
```

Comment:
I think you could potentially avoid some `app.update`s by using `elif autoscaling_is_paused() and is_autoscaling_resumed(app.item)` but I'm not sure.

Reply:
Yeah I think we can put them all in one line, but I'm afraid that is confusing to read. Also, I've condense this further by wrapping the whole thing with `if app` as opposed to writing it on each line.